### DISCUSSION TITLE
[Question] Comparison with emacs-reformatter and emacs-format-all-the-code

### DISCUSSION THREAD

**Discussion by doolio** [radian-software/apheleia#96] [Q&A] [answered]
This is not an issue rather a question for my own understanding so feel free to close immediately. How does this package compare to [purcell/emacs-reformatter](https://github.com/purcell/emacs-reformatter) which is itself an alternative to [lassik/emacs-format-all-the-code](https://github.com/lassik/emacs-format-all-the-code)? Thanks for your time.

**Comment by raxod502** [comment, marked as answer]
I actually have a great summary of this [in a spreadsheet](https://docs.google.com/document/d/1bIURUdHqlkF8QfFDnOP4ZOHXADkEtB_mbzMVoBQEBSw/edit)! See https://github.com/lassik/emacs-format-all-the-code/issues/170 for more discussion.

**Reply by doolio** [reply]
This is great. Thanks for your efforts. An additional point for consideration is whether each package is a dependency of another. I believe emacs-reformatter is a dependency of a few packages.

**Reply by raxod502** [reply]
Oh interesting, I didn't know that. Added to the spreadsheet, thanks.
